src/core/CubeCamera.tsx

```diff
@@ -1,64 +1,38 @@
-import { HalfFloatType, Fog, FogExp2, Group, Texture, CubeCamera as CubeCameraImpl, WebGLCubeRenderTarget } from 'three'
+import { Group, Texture } from 'three'
 import * as React from 'react'
-import { useFrame, useThree } from '@react-three/fiber'
+import { useFrame } from '@react-three/fiber'
+
+import { useCubeCamera, Props as HookProps } from './useCubeCamera'
 
 type Props = JSX.IntrinsicElements['group'] & {
-  /** Number of frames to render, Infinity */
-  frames?: number
-  /** Resolution of the FBO, 256 */
-  resolution?: number
-  /** Camera near, 0.1 */
-  near?: number
-  /** Camera far, 1000 */
-  far?: number
-  /** Custom environment map that is temporarily set as the scenes background */
-  envMap?: THREE.Texture
-  /** Custom fog that is temporarily set as the scenes fog */
-  fog?: Fog | FogExp2
   /** The contents of CubeCamera will be hidden when filming the cube */
   children: (tex: Texture) => React.ReactNode
-}
+  /** Number of frames to render, Infinity */
+  frames?: number
```

Comment:
I didn't include this `frames` prop, since limiting `update()` calls with `useCubeCamera` is trivial